README.md

```diff
@@ -1,11 +1,100 @@
-(TODO: actual README content)
+Node.js client for [FaunaDB](https://faunadb.com).
 
-## Build
+View documentation [here](https://faunadb.github.io/faunadb-js/).
+
+See the [FaunaDB Documentation](https://faunadb.com/documentation) for
+a complete API reference, or look in
+[`/tests`](https://github.com/faunadb/faunadb-python/tree/master/tests) for more
+examples.
+
+## Using the Client
+
+### Install
+
+  npm install faunadb/faunadb-js
+
+### Use
+
+All work with FaunaDB happens through a Client instance.
+All Client methods return [Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
+Although it's possible to use Client alone, it's often easier to do work through Model classes.
+For advanced work, the query api (`faunadb/lib/query`) will be useful.
+
+    import Client from 'faunadb/lib/client'
+    import {NotFound} from 'faunadb/lib/errors'
+    import Model from 'faunadb/lib/model/Model'
+    import {Class, Database, Key} from 'faunadb/lib/model/Builtin'
+    import {Ref} from 'faunadb/lib/objects'
+    import * as query from 'faunadb/lib/query'
+
+    const rootClient = new Client({secret: {
+      // Use your faunadb.com login here
+      user: '',
+      pass: ''
+    }})
+
+    async function testPing() {
+      console.log(await rootClient.ping())
+    }
+
+    async function createDBAndGetClient() {
+      const dbName = 'prydain'
+      const dbRef = new Ref('databases', 'prydain')
+
+      try {
+        await rootClient.delete(dbRef)
```

Comment:
I have an ES5 version, use that alone then? (no 'import', 'async', arrow functions)